server/paymentProviders/stripe/index.js

```diff
@@ -206,13 +207,22 @@ export default {
     }
   },
 
-  webhook: requestBody => {
+  webhook: request => {
+    const requestBody = request.body;
+
+    debug('stripe event triggered', requestBody.id);
```

Comment:
I was more thinking about a FULL log of the event `JSON.stringify(requestBody)`;

```suggestion
    debug(`Stripe Webhook Event received: ${JSON.stringify(requestBody)}`);
```